azure-pipelines.yml

```diff
@@ -79,18 +81,30 @@ stages:
             Write-Host "##vso[task.setvariable variable=SC_VERSION;isOutput=true]$version"
 
             Rename-Item -Path "$env:AGENT_BUILDDIRECTORY/a/sonar-scanner-vsts-0.0.2-SNAPSHOT-sonarcloud.vsix" -NewName $extensionNewName
+
+            $version = Get-AzureDevopsTaskVersion "$env:BUILD_SOURCESDIRECTORY/build/extensions" "sonarcloud" "analyze" "v1"
+            Write-Host "##vso[task.setvariable variable=SC_V1_ANALYZE_VERSION;isOutput=true]$version"
+
+            $version = Get-AzureDevopsTaskVersion "$env:BUILD_SOURCESDIRECTORY/build/extensions" "sonarcloud" "prepare" "v1"
+            Write-Host "##vso[task.setvariable variable=SC_V1_PREPARE_VERSION;isOutput=true]$version"
+
+            $version = Get-AzureDevopsTaskVersion "$env:BUILD_SOURCESDIRECTORY/build/extensions" "sonarcloud" "publish" "v1"
+            Write-Host "##vso[task.setvariable variable=SC_V1_PUBLISH_VERSION;isOutput=true]$version"
         pwsh: true
     - task: PublishBuildArtifacts@1
       displayName: 'Publish artifacts on the build'
       inputs:
         PathtoPublish: '$(Agent.BuildDirectory)/a'
         ArtifactName: 'extensions'
 - stage: unpublish
```

Comment:
Presumably this is un-publishing the "test" version of the artefact? If so, it would be clearer if the stage name/display name made it clear it's referring to the test version.
Same of the "publish" stage below.

Reply:
Yes is it, ok noted.

Reply:
Names have been modified